renderer/components/nettests/performance/Dash.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import {
+  Flex,
+  Box,
+  Text
+} from 'ooni-components'
+import { FormattedMessage } from 'react-intl'
+
+import StatusBox from '../../measurement/StatusBox'
+import formatBitrate from '../../formatBitrate'
+import formatSpeed from '../../formatSpeed'
+import performanceTestGroup from './index'
+
+const Dash = ({measurement, render}) => {
+  const testKeys = JSON.parse(measurement.test_keys)
+  const {
+    connect_latency,
+    median_bitrate,
+    min_playout_delay
+  } = testKeys
```

Comment:
what is this doing?

Reply:
It is called [object destructuring assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring). Equivalent to:
```
const connect_latency = testKeys.connect_latency
...
```

Reply:
oh, wow!